src/settings/AcquisitionUnits/AcquisitionUnitsList/AcquisitionUnitsList.js

```diff
@@ -29,25 +31,38 @@ const AcquisitionUnitsList = ({ acquisitionUnits, getViewPath, getCreatePath })
     history.push(getViewPath(id))
   ), [getViewPath, history]);
 
+  const shortcuts = [
+    {
+      name: 'new',
+      handler: () => history.push(getCreatePath()),
```

Comment:
Please, check permissions for create/edit

Reply:
updated